### DISCUSSION TITLE
New Permissions and User Perception

### DISCUSSION THREAD

**Discussion by thepete86** [tprouvot/Salesforce-Inspector-reloaded#326] [General]
Just wanted to bring to your attention that likely a good amount of users won't enable the latest update, due to this message:
"Salesforce Inspector reloaded is disabled
To re-enable it, accept the new permissions:
• Read and change your data on all sformapps.cn sites and all
sformproducts.cn sites."

I checked the Pull Request and understand why this is necessary, but to people who won't do that it looks very conspicuous that you would ever see this message, especially considering headlines with China and espionage, so there will likely be a good chunk of users who stop using this extension. At the same time, not that many people are that diligent so might not be that large of an amount of people. Just wanted to give my two cents on the topic, since I'm an avid user of this great extension and hope it keeps going for a while.

Seems someone has also made a review on the chrome extension store to that effect.

**Comment by tprouvot** [comment]
Hi @thepete86,
You are right, the addition of new domains made Chrome disable the extension until the new permissions are accepted.

Thank you for bringing my attention to the chrome review, I could answer thanks to you.
I hope they will check the banner and read the release notes to see the China new domains mention 🤞 

Hope you'll enjoy the new features !
